File: SampleProjects/2.1/NavigationPractice/NavigationPractice/NextViewController.m
//
//  NextViewController.m
//  NavigationPractice
//
//  Created by yuichi.takeda on 2/5/15.
//  Copyright (c) 2015 mixi, Inc. All rights reserved.
//

#import "NextViewController.h"

@interface NextViewController ()
@property (strong, nonatomic) IBOutlet UILabel *label;

@end

@implementation NextViewController

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
    self.label.text = self.messageFromPrevious;
}

@end

File: Swift/after/day2/1.1/NavigationSample/NavigationSample/NextViewController.swift
//
//  NextViewController.swift
//  NavigationSample
//
//  Created by 鈴木 大貴 on 2016/10/23.
//  Copyright © 2016年 szk-atmosphere. All rights reserved.
//

import UIKit

class NextViewController: UIViewController {

    @IBOutlet weak var label: UILabel!
    
    var buttonString: String?
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        guard let buttonString = buttonString else { return }
        label.text = "Button \(buttonString) Tapped"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
}
